.github/workflows/unit-tests.yml

```diff
@@ -21,38 +21,27 @@ jobs:
   build_test:
     strategy:
       matrix:
-        os: ["ubuntu-20.04", "macos-latest"]
+        os: ["ubuntu-20.04", "macos-11"]
         python: ["3.8", "3.9", "3.10", "3.11"]
 
     runs-on: ${{ matrix.os }}
 
     steps:
     - uses: actions/checkout@v3
-    - name: Set up Python ${{ matrix.python }}
+    - name: Set up Python 3.8
```

Comment:
Is there a specific reason for using `3.8` but not latest or something else?

Reply:
I picked 3.8 to make sure that no standard library functions introduced in later versions are used.